cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_log_files.json

```diff
@@ -156,6 +156,23 @@
       ],
       "feature_conditions": []
     },
+    {
+      "timestamp_format_key": "default",
```

Comment:
Is this `timestamp_format_key` manually tested? Asking because the console_output.logs are from system messge. And system message are in the format of
```
"timestamp_format_key": "month_first",
```

Reply:
I believe it is the same as the time format as clustermgtd - this entry is picking up the log file written out by a python logger that I've defined alongside clustermgtd's root logger and I've specified the format to be the same as clustermgtd.
So CloudWatch Agent will be parsing the event as written by clustermgtd and not parsing the contents (which is the system output) I think?

Reply:
ok sounds good, let's test it with the node package PR